bluesky/callbacks/best_effort.py

```diff
@@ -101,9 +101,9 @@ def start(self, doc):
                  "combine streams.")
         self.dim_fields = [f
                            for field, stream_name in dimensions
-                               for f in field]
-        _, self.dim_stream  = dimensions[0]
-    
+                           for f in field]
```

Comment:
You suggested this indentation when we pair-coded this part. You pointed out that it correctly illustrates the nested-ness of the for loop.